build/azure-pipeline.stable.yml

```diff
@@ -18,6 +18,8 @@ extends:
   template: azure-pipelines/extension/stable.yml@templates
   parameters:
     publishExtension: true
+    locTsConfigs: $(Build.SourcesDirectory)/tsconfig.json
+    locBundleDestination: $(Build.SourcesDirectory)/out
```

Comment:
As a reminder for future reviews of this PR, this should match the output `path:` in the webpack config: https://github.com/microsoft/vscode-jupyter/blob/main/build/webpack/webpack.extension.node.config.js#L142